README.md

```diff
@@ -60,3 +60,4 @@ documentation-best-practices
 
 [Zola]: https://www.getzola.org/
 [Abridge theme]: https://abridge.pages.dev/overview-abridge/
+[^1]: Included as a git submodule, so clone recursively or `git submodule update --init --recursive`.
```

Comment:
Hmm is it wise to put that at the bottom? That would warrant a line that says `git clone --recursive …`, no?

Reply:
Happy to change it as you suggest but I went with a footnote because I wanted to keep the local installation section short.

Reply:
I understand what you mean, but I'd rather put this on the forefront so that the first thing that people see is the correct invocation of `git`, and they don't have to check footnotes. :)

Reply:
I understand what you mean, but I'd rather put this on the forefront so that the first thing that people see is the correct invocation of git, and they don't have to check footnotes. :)

Reply:
I've moved the git submodule mention inline and included a complete getting started command sequence.